package.json

```diff
@@ -86,7 +86,7 @@
     "plaintest": "mocha --exit -t 10000 -b -R spec test/index.js && npm run tape",
     "prepublish": "npm run babel",
     "pre_test": "npm run lint",
-    "bin_test": "jake -f test/jake/*",
+    "bin_test": "jake -f test/jake/migrate.js",
```

Comment:
at least locally it doesn't execute anything for me unless specific file (or directory with file called "Jakefile") is specified.

Reply:
Maybe that is platform dependent what it does (shell might expand those wildcards which may break it). I remember having to put wildcard notations in quotes at least with mocha to make globs to work. Like `-f "test/jake/*"`

Reply:
You are probably right, I've checked logs for original PR that introduced CLI tests, and they do indeed show
```
> jake -f test/jake/*
☑ Create a migration file
☑ Run migrations
```

So I'll revert this change.